Sources/MongoSwift/ReadConcern.swift

```diff
@@ -14,65 +14,94 @@ public class ReadConcern: Codable {
         case linearizable
         /// See https://docs.mongodb.com/master/reference/read-concern-snapshot/
         case snapshot
-    }
-
-    /// A pointer to a `mongoc_read_concern_t`.
-    internal var _readConcern: OpaquePointer?
+        case custom(level: String)
```

Comment:
hmm, I'm not sure whether I agree with adding a `custom` case here. I think it implies that a custom level is some feature MongoDB has, when really this is just a catch-all way to provide a value that the server supports but our driver didn't add to the enum yet. Was there a reason for not just leaving the `level` as a `String?` ?

maybe the issue is just the name - maybe something like `other` might better communicate what this case is for?

we should also add a docstring to this case.

it may also be worth seeing how other drivers that provide an enum for the read concern level handle the unknown case.